test-cargo-miri/run.local_crate.stderr.ref

```diff
@@ -0,0 +1,26 @@
+subcrate,issue_1567,exported_symbol_dep,cargo_miri_test,cdylib,executable,library_with_dashes,exported_symbol,issue_1691,issue_1705,issue_rust_86261,proc_macro_crate
+error: Undefined Behavior: entering unreachable code
+ --> /build/local-crate/library-with-dashes/src/lib.rs:6:14
```

Comment:
Where does this path come from? It looks like an absolute path, which would be a problem -- the test is meant to work everywhere, not just on CI.

Also, this fails on Windows because there, some of the `/` become `\`.

Our test-cargo-miri isn't really set up to deal with error output I am afraid. Maybe it'd be easier to just test the contents of `MIRI_LOCAL_CRATES` rather than an actual error message?

Reply:
> Where does this path come from? It looks like an absolute path, which would be a problem -- the test is meant to work everywhere, not just on CI.

The test runner uses `--remap-path-prefix` to replace the working directory with `/build`: https://github.com/rust-lang/miri/pull/3644/commits/8bb14dc988abe511307afd07a3ec9427ca1f87f2#diff-4fcb5764801672d83a446351eb4d6f44577f99d386d2632964e7a239bfbbb813R137

> Maybe it'd be easier to just test the contents of `MIRI_LOCAL_CRATES` rather than an actual error message?

Yeah, that’s probably better.